src/sync/discovery.ts

```diff
@@ -38,18 +39,18 @@ export const getRepositoryTask = async (
 	let edges: RepositoryNode[];
 	let repositories: Repository[];
 	if (await booleanFlag(BooleanFlags.USE_REST_API_FOR_DISCOVERY, jiraHost)) {
```

Comment:
Looks like the FF is off, which means the modified code will never fire in prod. However, tested it locally just in case

<img width="200" alt="image" src="https://user-images.githubusercontent.com/20631664/228083659-9d5e1db4-d447-43fb-8eaa-1561dab81b72.png">